sdk/mixedreality/azure-mixedreality-authentication/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.1.1 (2021-09-13)
 
+### Other changes
+
+Updated dependencies.
```

Comment:
We can update this to: 
### Other changes
#### Dependency updates
 - upgraded azure-core dependency to version 1.20.0